lib/SILOptimizer/Mandatory/DIMemoryUseCollectorOwnership.cpp

```diff
@@ -774,10 +774,14 @@ void ElementUseCollector::collectUses(SILValue Pointer, unsigned BaseEltNo) {
       continue;
     }
 
-    if (isa<LoadWeakInst>(User)) {
-      trackUse(DIMemoryUse(User, DIUseKind::Load, BaseEltNo, 1));
-      continue;
+#define NEVER_LOADABLE_CHECKED_REF_STORAGE(Name, ...) \
+    if (isa<Load##Name##Inst>(User)) { \
+      trackUse(DIMemoryUse(User, DIUseKind::Load, BaseEltNo, 1)); \
+      continue; \
     }
+#define SOMETIMES_LOADABLE_CHECKED_REF_STORAGE(Name, ...) \
+  NEVER_LOADABLE_CHECKED_REF_STORAGE(Name, "...")
+#include "swift/AST/ReferenceStorage.def"
```

Comment:
I would expect all the checked storage types to be present here.

Reply:
I've fixed this in a forthcoming pull request update.